browser/net/url_context.cc

```diff
@@ -108,9 +106,7 @@ std::shared_ptr<brave::BraveRequestInfo> BraveRequestInfo::MakeCTX(
   // (See |BraveProxyingWebSocket|).
   if (ctx->tab_origin.is_empty()) {
     ctx->tab_origin = brave_shields::BraveShieldsWebContentsObserver::
-                          GetTabURLFromRenderFrameInfo(ctx->render_process_id,
-                                                       ctx->render_frame_id,
-                                                       ctx->frame_tree_node_id)
+                          GetTabURLFromRenderFrameInfo(ctx->frame_tree_node_id)
```

Comment:
also a possible follow-up is to get rid of this call and just get webcontents and its `LastCommittedURL` right there

Reply:
i was about to do this at some point https://github.com/brave/brave-core/pull/5925/files

Reply:
It seems you'd need `content::WebContents::FromFrameTreeNodeId()` for that as well and I'm again not sure of what would happen if no valid WebContents was to be returned (maybe that can't happen, though), so I'd rather leave this unchanged for now if it's ok.